docker/k8s/sgx-enclave-manager-deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: enclave-manager-pool
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: tee
+  template:
+    metadata:
+      name: tee
+      labels:
+        app: tee
+    spec:
+      containers:
+      - image: tcf-dev:latest
+        name: tee
+        imagePullPolicy: Never
+        volumeMounts:
+        - mountPath: /dev/isgx
+          name: isgx
+        - mountPath: /var/run/aesmd/aesm.socket
+          name: aesmsocket
+        securityContext:
+          privileged: true
+        envFrom:
+        - prefix: TCF_
+          configMapRef:
+            name: tcf-config
+        command: ["/bin/bash"]
+        args: ["$(TCF_HOME)/scripts/enclave_manager.sh"]
```

Comment:
Is it a requirement to set TCF_HOME before running this yaml file? If so, can we mention this in documentation?

Reply:
This is set in https://github.com/hyperledger/avalon/blob/master/docker/k8s/tcf-config-map.yaml. Documentation mentions initializing the config map.

Reply:
Okay